app/enterprise/changelog.md

```diff
@@ -4,6 +4,167 @@ no_search: true
 no_version: true
 skip_read_time: true
 ---
+## 2.3.0.0
+**Release Date** 2020/02/10
+
+### **Kong Gateway (Enterprise)**
+The following sections list {{site.ee_product_name}}-exclusive updates,
```

Comment:
I'm curious why we use template placeholders here vs a specific product name in the line above?  I'm also curious: will the changelog end up changing if the template placeholders change e.g. in 2.5.x or 3.0?

Reply:
Good point - maybe we shouldn't use variables in the changelog at all, since it's supposed to be a snapshot in time.

We generally don't use them in headers or titles for linking purposes.